index.js

```diff
@@ -200,6 +200,275 @@ class Client {
+                if (err) return reject(err);
+
+                resolve(data.data);
+            });
+        });
+
+    }
+
+    // https://api-dev.thetvdb.com/swagger#!/Series/get_series_id_filter_params
+
+    getSeriesFilterParam(id) {
+        let self = this;
+
+        return new Promise(function(resolve, reject) {
+            self.request.get({
+                uri: `series/${id}/filter/params`,
+                headers: {
+                    "Authorization": `Bearer ${self.token}`,
+                    "Accept-Language": self.language
+                }
+            }, function(err, res, data) {
+                if (err) return reject(err);
+
+                resolve(data.data);
+            });
+        });
+    }
+
+    // https://api-dev.thetvdb.com/swagger#!/Series/get_series_id_images
+
+    getSeriesImages(id) {
+        let self = this;
+
+        return new Promise(function(resolve, reject) {
+            self.request.get({
+                uri: `series/${id}/images`,
+                headers: {
+                    "Authorization": `Bearer ${self.token}`,
+                    "Accept-Language": self.language
+                }
+            }, function(err, res, data) {
+                if (err) return reject(err);
+
+                resolve(data.data);
+            });
+        });
+    }
+
+    // https://api-dev.thetvdb.com/swagger#!/Series/get_series_id_images_query
+    // TO-DO: verify how to send query params
+    // keyType
+    // resolution
+    // subKey
+    getSeriesImagesQuery(id, params) {
+        let self = this;
+        let query = params;
+
+        return new Promise(function(resolve, reject) {
+            self.request.get({
+                uri: `series/${id}/images/query`,
+                headers: {
+                    "Authorization": `Bearer ${self.token}`,
+                    "Accept-Language": self.language
+                },
+                qs: query
+            }, function(err, res, data) {
+                if (err) return reject(err);
+
+                resolve(data.data);
+            });
+        });
+    }
+
+    // https://api-dev.thetvdb.com/swagger#!/Series/get_series_id_images_query_params
+
+    getSeriesImagesParams(id) {
+        let self = this;
+
+        return new Promise(function(resolve, reject) {
+            self.request.get({
+                uri: `series/${id}/images/query/params`,
+                headers: {
+                    "Authorization": `Bearer ${self.token}`,
+                    "Accept-Language": self.language
+                }
+            }, function(err, res, data) {
+                if (err) return reject(err);
+
+                resolve(data.data);
+            });
+        });
+    }
+
+    //Updates
+    // https://api-dev.thetvdb.com/swagger#!/Updates/get_updated_query
+
+    getUpdates(fromTime, toTime) {
+        let self = this;
+        let query = {};
+
+        query['fromTime'] = fromTime;
+        query['toTime'] = toTime;
```

Comment:
Like this? 

qs: {
                    "fromTime": fromTime,
                    "toTime": toTime
                }